gradle.properties

```diff
@@ -0,0 +1,60 @@
+modName = NEIIntegration
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = neiintegration
+customArchiveBaseName = NEIIntegration
```

Comment:
Please keep the file structure and comments from ExampleMod. This way it is much easier to navigate the configuration files. Also, people don't read manuals or anything like that. The comment has the best chance of being read. It should be kept!

Reply:
It was copied from NEI and added this to keep the output file the same.